packages/strapi-admin/admin/src/components/Row/index.js

```diff
@@ -30,13 +34,35 @@ class Row extends React.Component {
     this.props.onDeleteClick(e);
   }
 
+  renderImg = () => {
```

Comment:
You need to find a way to make this more dynamic, we need to update the backend of the marketplace to send you the necessary information to the front.